src/Makefile.3ds

```diff
@@ -200,6 +203,8 @@ endif
 ifneq ($(DEPSDIR),$(BUILD))
 $(DEPSDIR):
 	@mkdir -p $@
+$(DEPSDIR)/borg:
+     @mkdir -p $@/borg
```

Comment:
Yes, that's almost right, just change spaces before the "\@mkdir -p $\@/borg" to a single tab: that's what make expects there.